.github/workflows/solutions-dpg-qa.yaml

```
@@ -0,0 +1,156 @@
+permissions:
+  id-token: write
+  contents: write
+jobs:
+  check-changed-files:
+    runs-on: ubuntu-latest
+    outputs:
+      deploy-as-code-changed: ${{ steps.check_files.outputs.deploy-as-code-changed }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)